python/torch_mlir/extras/fx_importer.py

```diff
@@ -125,6 +135,8 @@
     func as func_dialect,
 )
 
+from .._mlir_libs._torchMlir import get_int64_max, get_int64_min
```

Comment:
We're running into issues with this line of code downstream: https://github.com/iree-org/iree/pull/18388#discussion_r1736600593

The relative path (into a directory that isn't a full package with a `__init__.py`? I'm not sure) is showing up as missing: 
```
wheelhouse/test.venv/lib/python3.11/site-packages/shark_turbine/aot/support/ir_utils.py:16: in <module>
    from iree.compiler.extras.fx_importer import (
wheelhouse/test.venv/lib/python3.11/site-packages/iree/compiler/extras/fx_importer.py:138: in <module>
    from .._mlir_libs._torchMlir import get_int64_max, get_int64_min
E   ModuleNotFoundError: No module named 'iree.compiler._mlir_libs._torchMlir'
```

I'm not sure if we're missing some setup downstream to make this work, but do you really need to jump over to C++ to get two integer values? Can you use [`sys.maxsize`](https://docs.python.org/3/library/sys.html#sys.maxsize) or `2**63 - 1` for the max and something equivalent for the min? Or some other sentinel value as needed.

Reply:
One issue with `sys.maxsize` is it'd be platform dependent (`2**31 - 1` on 32-bit vs `2**63 - 1` on 64-bit), possibly causing spurious behavior (depending on where it is run), and/or deviation from PT. Hardcoding it sounds OK to unblock (unless there's a better way to address the relative path issue downstream).

Reply:
I think our downstream use is missing this: https://github.com/llvm/torch-mlir/blob/98e08023bbf71e00ab81e980eac9f7c96f1f24b4/setup.py#L223-L225

(here's our current list: https://github.com/iree-org/iree/blob/3ceffb60490107ae76a9c20b8a1588d560be0218/compiler/setup.py#L432-L441)

I'm not sure if we want to take on that dependency though, especially for two integers. @stellaraccident any ideas?

Reply:
How about using something like `torch.iinfo(torch.int64)`<https://pytorch.org/docs/stable/type_info.html>?

Reply:
That would work. Thanks for sending the fix!

Reply:
For the future: this file must be self contained. If I'd caught this, I would have commented when the patch was out up. If this has already been fixed forward, fine. Otherwise, please revert and reapply without the native code changes.